### ISSUE TITLE
Breaks on Android 5.1.1

### ISSUE THREAD

**Issue by p-sebastian** [gbenvenuti/cordova-plugin-screen-orientation#62]
I have tested it on an emulator, and an actual phone on android Lollypop 5.1.1, and It just stops working completely. 
tried:
screen.lockOrientation('ORIENTATION_STRING');
unlockOrientation();

**Comment by gbenvenuti** [comment]
Hi,

I'm going to need more information here, do you have more information, i.e. console/logcat output?

Also please provide some javascript where you are using the calls.  Within `deviceReady`.  I'm assuming that `unlockOrientation()` is a typo?

**Comment by redbird19** [comment]
Hi,

I have the same problem.  Everything works fine on my Android phone which is version 4.4.2, but a user on Samsung Galaxy S6 Edge with Android 5.0.2 reported that my app wasn't functioning properly.  They were nice enough to let me (quickly) install a "debug" version of my app to narrow down the problem to the following lines of code:

if (screen.lockOrientation && screen.orientation) {
    screen.lockOrientation('portrait');
}

I got around my particular problem by putting a try-catch block around this section of code and letting it fail silently, which was an acceptable (albeit temporary) workaround for me, as enforcing the screen orientation to portrait isn't critical to my app, but it would be nice to get this fixed.  Once I added the try-catch block the user was able to use my app without issue (although it was no longer locked in portrait mode).

Unfortunately, I don't have access to an Android 5.0+ device any more, so I can't provide much more information.  But, interestingly, I went to a mobile store and attempted to test the old version of my app on a Samsung Galaxy S6 Edge with Android 5.0.2, and was unable to reproduce the problem, which has left me baffled.  Maybe it's Lollipop in combination with some system setting?
